packages/config/config/devices/0x044c/myot4.json

```diff
@@ -0,0 +1,112 @@
+{
+	"manufacturer": "SmartDHOME",
+	"manufacturerId": "0x044c",
+	"label": "MyOT v4",
+	"description": "OpenTherm Actuator v4",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0004"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "31",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Summer Mode",
+			"description": "Turn heating on/off"
```

Comment:
This feels redundant. Maybe just make the label `"Heating"` and remove the description?
Need to take care though that the values for Enable and Disable are not inverted then.

Reply:
Yes, it can be confusing, to turn off the heating you must turn it on. However, I prefer to keep the original names from the documentation. Honestly, it is really annoying when I can't find the parameters of well known device because of changed names. I have to find them by numbers. Unfortunately, renaming parameters is common practice in this project. This change makes myot.json and myot2.json inconsistent. The decision is yours.

Reply:
I guess consistency with the other similar devices is more important.

Reply:
As for the renamed labels: Some are really f-ing terrible. We prefer them to be simple and self-explanatory. Needing a manual to understand what a parameter does, isn't nice, especially when manuals get out of date due to firmware updates etc.
I understand that it can be sometimes hard to match them to the manual this way, but that is something we're willing to accept.